src/transformers/models/gemma2/modeling_gemma2.py

```diff
@@ -317,7 +317,7 @@ def forward(
                 # In case we are beyond the sliding window, we need to correctly offset the mask slicing
                 offset = cache_position[-1] - effective_seq_len + 1
                 # Should only be used when beyond the sliding window (i.e. offset > 0)
-                offset = max(0, offset)
+                offset = torch.clamp(offset, min=0)
```

Comment:
There is PR to automatically do it: https://github.com/pytorch/pytorch/pull/151348